README.md

```
@@ -97,17 +97,34 @@ $ brew uninstall --cask csv2notion-neo
-$ cd csv2notion_neo/
-$ poetry install --no-dev
-$ poetry run csv2notion_neo
+$ cd csv2notion-neo/
+$ ./scripts/local-test-build.sh
+$ ./test-build/csv2notion_neo --help
+```
+
+To run from source without building the binary (after the script has installed dependencies):
+
+```shell
+$ .build/venv/bin/csv2notion_neo --help
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)